docs/_docs/templates.md

```diff
@@ -421,56 +421,164 @@ The default is `default`. They are as follows (with what they filter):
-your site, you can use the `include` tag.
+If you have small page fragments that you want to include in multiple places on your site, you can use the `include` tag:
 
 ```liquid
 {% raw %}{% include footer.html %}{% endraw %}
 ```
 
-Jekyll expects all include files to be placed in an `_includes` directory at the
-root of your source directory. This will embed the contents of
-`<source>/_includes/footer.html` into the calling file.
+Jekyll expects all include files to be placed in an `_includes` directory at the root of your source directory. This will embed the contents of `<source>/_includes/footer.html` into the calling file.
 
-<div class="note">
-  <h5>ProTip™: Use variables as file name</h5>
-  <p>
+### Including files relative to another file
 
-    The name of the file you wish to embed can be literal (as in the example above),
-    or you can use a variable, using liquid-like variable syntax as in
-    <code>{% raw %}{% include {{my_variable}} %}{% endraw %}</code>.
+You can also choose to include file fragments relative to the current file by using the `include_relative` tag:
 
-  </p>
-</div>
+```liquid
+{% raw %}{% include_relative somedir/footer.html %}{% endraw %}
+```
+
+You won't need to place your included content within the `_includes` directory. Instead,
+the inclusion is specifically relative to the file where the tag is being used. For example,
+if `_posts/2014-09-03-my-file.markdown` uses the `include_relative` tag, the included file
+must be within the `_posts` directory, or one of its subdirectories. 
+
+Note that you cannot use the `../` syntax to specify an include location that refers to a higher-level directory.
+
+All the other capabilities of the `include` tag are available to the `include_relative` tag,
+such as using variables.
+
+### Using variables names for the include file
+
+The name of the file you want to embed can be specified as a variable instead of an actual file name. For example, suppose you defined a variable in your page's frontmatter like this:
 
-You can also pass parameters to an include. Omit the quotation marks to send a variable's value. Liquid curly brackets should not be used here:
+```yaml
+---
+title: My page
+my_variable: footer_company_a.html
+---
+```
+
+You could then reference that variable in your include:
+
+```liquid
+{% raw %}{% include {{page.my_variable}} %}{% endraw %}
+```
 
+In this example, the include would insert the file footer_company_a.html from the \_includes directory.
+
+### Passing parameters to includes
+
+You can also pass parameters to an include. For example, suppose you have a file called note.html in your \_includes folder that contains this formatting: 
+ 
 ```liquid
-{% raw %}{% include footer.html param="value" variable-param=page.variable %}{% endraw %}
+{% raw %}<div markdown="span" class="alert alert-info" role="alert">
+<i class="fa fa-info-circle"></i> <b>Note:</b> {{include.content}}
+</div>{% endraw %}
```

Comment:
I moved the treatment of variable parameters into another section titled "Passing parameter variables to includes". My new example here shows a different syntax. The code is right (not munged or anything).